camera_matrix: [1053.277717819183,  0.000000,           960.5,      0.000000,
                0.000000,           1053.277717819183,  600.5,      0.000000,
                0.000000,           0.000000,           1.000000,   0.000000]


distortion: [0.0, 0.0, 0.0, 0.0, 0.0]

# L_p = L_R_C * C_p + L_t_C

# L_R_C_ROS
rlc: [1.000000, -0.000930, -0.000358,
      0.000930, 1.000000, 0.000418,
      0.000358, -0.000418, 1.000000]

# L_t_C_ROS
tlc: [-0.530137, 0.589015, -0.737218]